README.rst

```diff
@@ -15,8 +16,39 @@ unasync
     :alt: Coverage Status
     :target: https://codecov.io/gh/RatanShreshtha/unasync
 
-Welcome to `unasync <Project URL (for setup.py metadata)>`__!
+.. |documentation| image:: https://readthedocs.org/projects/unasync/badge/?version=latest
+    :alt: Documentation Status
+    :target: https://unasync.readthedocs.io/en/latest/?badge=latest
+
+
+Welcome to `unasync <https://pypi.org/project/unasync/>`_, a project that can transform your asynchronous code into synchronous code.
+
+Installation
+============
+
+::
+
+    pip install unasync
+
+Usage
+=====
+
+To use the unasync project you need to install the package and then create a **_async** where you will place the asynchronous code you want to transform into synchronous code.
+
+And then in your :code:`setup.py` place the following code.
+
+.. code-block:: python
+    import unasync
+
+    setuptools.setup(
+        ...
+        cmdclass={'build_py': unasync.build_py},
+        ...
+    )
+
+And when you will build your package you will get your synchronous code in **_sync** folder.
 
-The async transformation code.
+Documentation
+=============
 
-License: Your choice of MIT or Apache License 2.0
```

Comment:
Can you please keep the license? The users can choose between MIT or Apache License 2.0, so I guess that's an interesting thing to say.

Reply:
Okay